src/tf_individual_transform_chain.rs

```diff
@@ -60,12 +60,12 @@ impl TfIndividualTransformChain {
                 if x == 0 {
                     return Err(TfError::AttemptedLookupInPast(
                         time,
-                        self.transform_chain.first().unwrap().clone(),
+                        Box::new(self.transform_chain.first().unwrap().clone()),
```

Comment:
I want to know why we need `box` here.

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#result_large_err

> A `Result` is at least as large as the `Err`-variant. While we expect that variant to be seldomly used, the compiler needs to reserve and move that much memory every single time.